conda-installation.md

```diff
@@ -247,3 +269,71 @@ arguments = myscript.py
 transfer_input_files = Miniconda3-latest-Linux-x86_64.sh, script.py, other_input.file
 ```
 {:.sub}
+
+# Specifying Exact Dependency Versions
+
+An important part of improving reproducibility and consistency between runs
+is to ensure that you use the correct/expected versions of your dependencies.
+
+When you run a command like `conda install numpy`, `conda` tries to install
+**the most recent version of `numpy`**. For example, `numpy` version `1.18.2`
+was released on March 17, 2020. To install exactly this version of `numpy`, you
+would run `conda install numpy==1.18.2` (the same works for `pip`). We
```

Comment:
```suggestion
would run `conda install numpy=1.18.2` (the same works for `pip`). We
```
conda uses `=` and pip `==`? That's from memory, I didn't confirm.

Reply:
Ack, very good catch.